examples/flutter_gallery/lib/demo/cupertino/cupertino_switch_demo.dart

```diff
@@ -23,15 +23,60 @@ class _CupertinoSwitchDemoState extends State<CupertinoSwitchDemo> {
         title: const Text('Cupertino Switch'),
       ),
       body: new Center(
-        child: new CupertinoSwitch(
-          value: _switchValue,
-          onChanged: (bool value) {
-            setState(() {
-              _switchValue = value;
-            });
-          }
-        ),
-      ),
+        child: new Container(
+          height: 300.0,
+          child: new Column(
+            children: <Widget>[
+              new Expanded(
```

Comment:
You can just use the mainAxisAlignment on the column instead of wrapping each child